src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,12 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
             }
+
+            String direction = null;
+            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
+                direction = String.valueOf(informed.getTrip().getDirectionId());
```

Comment:
It's actually also stored as an int in the [new loader](https://github.com/opentripplanner/OpenTripPlanner/blob/master/src/main/java/com/conveyal/gtfs/model/Trip.java#L29), so it would probably be best to have it as the same type here also.